backend/api/communication/src/main/java/co/airy/core/api/communication/lucene/DocumentMapper.java

```diff
@@ -33,6 +34,12 @@ public Document fromConversationIndex(ConversationIndex conversation) {
             document.add(new TextField("tag_ids", tagId, Field.Store.YES));
         }
 
+        for (MetadataNode node : conversation.getMetadata()) {
+            final String key = String.format("metadata.%s", node.getKey());
+            // Index but don't store metadata
+            document.add(new TextField(key, node.getValue(), Field.Store.NO));
```

Comment:
We can't do range queries on unstructured metadata, because we need to know the field type upfront (see `ExtendedQueryParser`)